airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -186,7 +186,7 @@ def delete_job(self, url: str):
 
     def next_page_token(self, last_record: dict) -> str:
         if self.primary_key and self.name not in UNSUPPORTED_FILTERING_STREAMS:
-            return f"WHERE {self.primary_key} > '{last_record[self.primary_key]}' "
+            return f"WHERE {self.primary_key} >= '{last_record[self.primary_key]}' "
```

Comment:
are PKs numeric and incremental? why are we not using cursor field here?

Reply:
This is used for `Full Refresh` streams. It turned out that it is impossible to use `Offset `more than `2000` - Salesforce API swears at this, so I had to use the same method as for Incremental, only use `ID`, not `cursor_field`.